### DISCUSSION TITLE
Please unlock educational benefits for my github education account

### DISCUSSION THREAD

**Discussion by ghousekhan23ee007** [community/community#145895] [GitHub Education]
### Hi everyone,

HI,
My request has been approved for education account on 27th of this month. Request you to please make me available the relevent benifits soon possible. Thank you.

**Comment by quoc10973** [comment]
You need to wait about 3 days

**Comment by davevad93** [comment]
Hi @ghousekhan23ee007 ,

GitHub no longer provide manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ghousekhan23ee007 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
